apps/dashboard/src/pages/VerifyEmail.tsx

```diff
@@ -13,7 +13,10 @@ const VerifyEmail: React.FC = () => {
     dispatch(findCurrentUser.call(atlas));
   }, [atlas, dispatch]);
   return (
-    <EmailVerification onVerification={onVerification} root={Route.Root} />
+    <DashboardEmailVerification
+      onVerification={onVerification}
+      root={Route.Root}
+    />
```

Comment:
No need to have the logic into seperate component we can write the component logic directly here.